### DISCUSSION TITLE
Empty resource config in asset launchpad

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17773] [Q&A] [answered]
Hi
Tell me please how can I see the configured config of resource in asset launchpad?

Why is it empty?

Code
```python
@dg.asset(
    group_name="databricks_bronze",
    resource_defs={
        "databricks_pyspark_step_launcher": databricks_pyspark_step_launcher.configured(DATABRICKS_PYSPARK_STEP_LAUNCHER_RESOURCE_CONFIG)
    },
)
```
![Screenshot 2023-11-07 at 15 10 50](https://github.com/dagster-io/dagster/assets/19783112/c093fb53-2182-494d-9d9e-41e4123402da)

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699366317700579?thread_ts=1699366317.700579&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment, marked as answer]
Hi there,

In dagster version 1.3 we introduced pythonic resources, which allow you to define resources using classes with pydantic style configuration. This vastly simplifies working with resources and allowed us to introduce features like displaying configuration in the UI. However, the `databricks_pyspark_step_launcher` wasn't updated to use the new system because of some technical challenges. I can open an issue to track this and get the discussion started again

**Reply by jamiedemaria** [reply]
https://github.com/dagster-io/dagster/issues/17787
